sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -285,7 +285,7 @@ static int sof_rt1015_hw_params(struct snd_pcm_substream *substream,
 
 	for_each_rtd_codec_dais(rtd, i, codec_dai) {
 		ret = snd_soc_dai_set_pll(codec_dai, 0, RT1015_PLL_S_BCLK,
-					  params_rate(params) * 50,
+					  params_rate(params) * 64,
```

Comment:
how do you generate 64*fs for the bitclock? which platform is this on?

Reply:
This is JSL+, my understanding is the blck is 64*48000=3.072Mhz.

Reply:
yes, but that requires the 24.576MHz PLL for the clock source, this is not available on CML so wanted to double-check.